spec/javascript/packages/document-capture/services/upload-spec.js

```diff
@@ -79,19 +81,19 @@ describe('document-capture/services/upload', () => {
   it('handles pending success', async () => {
     const endpoint = 'https://example.com';
 
-    sandbox.stub(window, 'fetch').callsFake((url, init) => {
+    sandbox.stub(global, 'fetch').callsFake((url, init) => {
       expect(url).to.equal(endpoint);
       expect(init.body).to.be.instanceOf(window.FormData);
       expect(init.body.get('foo')).to.equal('bar');
 
-      return Promise.resolve(
-        new Response(
-          JSON.stringify({
-            success: true,
-          }),
-          { status: 202, url: endpoint },
-        ),
```

Comment:
The second parameter ([ResponseInit](https://fetch.spec.whatwg.org/#responseinit)) doesn't support `url`. This worked previously in the browser implementation, but Node's native implementation won't respect it.

This would also have been caught if these files were TypeScript ([playground example](https://www.typescriptlang.org/play?#code/HYUw7gBASiDOAOB7YsQAoBQFsQFIGUB5AOQDpYAXAJwEtgBzGgMwE80BvLHb2AVwGN+cWAC4I1XiAA0XbAF8AlDO7sIlAIYVeoiACYADLqkReVADZiA5KRuWIcmQoDcQA)).